src/tests/test_magic.py

```diff
@@ -187,6 +187,23 @@ def test_persist_no_index(ip):
     assert persisted == [(1, "foo"), (2, "bar")]
 
 
+@pytest.mark.parametrize(
+    "sql_statement, expected_error",
+    [
+        ("%%sql --stuff\n SELECT * FROM test", "Unrecognized argument(s)"),
+        ("%%sql --unknown\n SELECT * FROM test", "Unrecognized argument(s)"),
+        ("%%sql --invalid-arg\n SELECT * FROM test", "Unrecognized argument(s)"),
```

Comment:
add some examples that begin with a single dash, e.g., `%%sql -Z`

Reply:
update it